gin/utils.go

```diff
@@ -104,6 +105,14 @@ func getStringFromSession(s sessions.Session, name string) string {
 	return ""
 }
 
+// helper func to extract the last number of the order string.
+// this will be referred to as the "friendly order number" and is the main way users will refer to the orders.
+// example: SO-VSSP-43 will returns 43
+func fromTagToNumber(tag string) string {
```

Comment:
@dpham273 created a new function here just to extract the last number from the tag. Feel free to create small, utility functions in `gin/utils.go` and then use it in the template